hashicorp/terraform-provider-tfe
# Add Microsoft Teams destination to notification_configuration

## Description

Adds `microsoft_teams` as a valid destination for a `notification_configuration`

The Microsoft Teams notification config will be the same as the Slack notification config, so this is mostly a copy of the Slack notification implementation and tests

Blocked on:
 - [ ] Feature completion
 - [ ] https://github.com/hashicorp/go-tfe/pull/389
 - [ ] Update vendored go-tfe version
 - [ ] Test output

## Testing plan

1.  Check that the `tfe` provider can create, update, and destroy notification configurations for MS Teams
2. Check that notification config resources tar-getting microsoft teams are correctly validated (same requirements as the Slack config)

## External links

- [Go TFE PR](https://github.com/hashicorp/go-tfe/pull/389)
- [Asana](https://app.asana.com/0/1201654039499505/1202142043071082/f)

## Output from acceptance tests

TODO blocked on a dev env issue
```
$ TESTARGS="-run TestAccTFEWorkspace" make testacc 

...
```
